src/projects/diff/diff.html

```diff
@@ -102,203 +101,39 @@
         },
         baseUrl: String,
         sha: String,
-        additions: {
-          type: Number,
-          value: 0
-        },
-        deletions: {
-          type: Number,
-          value: 0
-        },
-        diff: {
-          type: String,
-          observer: '_parseDiffToGroups'
-        },
-        groups: {
+        additions: Number,
+        deletions: Number,
+        diff: String,
+        hunks: {
           type: Array,
           value: function() {
             return [];
           }
         },
         orderedGroups: {
           type: Array,
-          computed: '_orderGroups(groups.*, ordering)'
-        },
-        ordering: {
-          type: Array,
-          value: function() {
-            return [];
-          },
-          observer: '_updateGroups'
+          observer: '_groupsOrdered'
         }
       },
-      _updateGroups: function() {
-        this.groups.forEach(function(e) {
-          delete e._index;
-        });
-        // this.notifyPath('groups', this.groups);
-      },
-      _addGroup: (function() {
-        var newGroup = {
-          file: 'New file',
-          diffId: 'new-group'
-        };
-
-        return function() {
-          if (!this.ordering.length || this.ordering[0][0] !== newGroup) {
-            this.ordering.unshift([
-              newGroup
-            ]);
+      _groupsOrdered: function(orderedGroups) {
```

Comment:
Some leftover debug statements and commented code in this function. Is that intended?